tests/app/views/contexts/test_submitted_response_context.py

```diff
@@ -65,14 +65,24 @@ def test_build_view_submitted_response_context_submitted_text(app: Flask):
 
 def test_build_view_submitted_response_context_submitted_text_social(app: Flask):
     with app.app_context():
-        questionnaire_store = fake_questionnaire_store()
+        questionnaire_store = fake_questionnaire_store_social()
         context = build_view_submitted_response_context(
             "en", SCHEMA, questionnaire_store, SurveyType.SOCIAL
         )
 
         assert context["submitted_text"] == "Answers submitted."
 
 
+def test_build_view_submitted_response_context_submitted_text_health(app: Flask):
+    with app.app_context():
+        questionnaire_store = fake_questionnaire_store_social()
+        context = build_view_submitted_response_context(
+            "en", SCHEMA, questionnaire_store, SurveyType.HEALTH
+        )
+
+        assert context["submitted_text"] == "Answers submitted."
```

Comment:
Is it worth updating `test_theme_health` so that view submitted response is enabled so there is a test schema demoing this?

Reply:
Added, thanks!